### ISSUE TITLE
Improve Accessibility / Talkback

### ISSUE THREAD

**Issue by Bertware** [hyperrail/hyperrail-for-android#8]
The application should be usable to users who are using accessibility tools, like talkback.

- Check if navigation with talkback works as expected (should be clear, easy to use)
- Improve or add contentDescriptors where required
